corehq/apps/domain/templates/domain/select_plan.html

```diff
@@ -117,6 +129,37 @@ <h4 class="text-center">
       </a>
     </div>
 
+    <div class="alert alert-warning text-center"
+         data-bind="visible: oIsNextPlanPaused">
+      <i class="fa fa-warning"></i>
+      {% blocktrans %}
+        Your subscription will be pausing on
+        <span data-bind="text: oStartDateAfterMinimumSubscription"></span> unless
+        you select a different plan above.
+      {% endblocktrans %}
+    </div>
+
+    <div class="alert alert-warning text-center"
+         style="margin-top: 20px;"
+         data-bind="visible: oIsNextPlanDowngrade">
+      <i class="fa fa-warning"></i>
+      {% blocktrans %}
+        Your subscription will be downgrading to
+        <span data-bind="text: oNextSubscription"></span> on
+        <span data-bind="text: oStartDateAfterMinimumSubscription"></span> unless
+        you select a different plan above.
+      {% endblocktrans %}
+    </div>
+
+    <div class="community-plan-notice alert alert-info text-center"
+         data-bind="visible: oIsCurrentPlanCommunity">
+      <p class="lead">
+        {% blocktrans %}
+          You are currently on the FREE CommCare Community plan.
```

Comment:
I'm not quite following why this is still here, isn't community gone?

Reply:
community still exists. you can immediately sign up for a community plan as an option from the start, and you can upgrade from community up. you just can't downgrade to community.